src/internet/radiobrowser/radiobrowserservice.cpp

```diff
@@ -419,9 +420,28 @@ void RadioBrowserService::ResolveStationUrl(const QUrl& original_url) {
     reply->deleteLater();
     QJsonDocument document = ParseJsonReply(reply);
 
-    QJsonObject item = document.object();
+    StreamList list;
+    QJsonArray contents = document.array();
+    QJsonValue item = contents.first();
+    if (item.isUndefined()) return;
```

Comment:
Should the url handler emit AsyncLoadComplete with LoadResult::NoMoreTracks in this case? (That might invalidate my suggestion that we use a single signal.)

Reply:
Right, this is an error, fixed